CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 #### Features
 
+* [#1393](https://github.com/ruby-grape/grape/pull/1393): Middleware can be inserted before or after default Grape middleware - [@ridiculous](https://github.com/ridiculous).
```

Comment:
I think the CHANGELOG entry below covers this functionality, I would just add `, [#1393]` to it. No big deal.